modules/arrow/src/types.ts

```diff
@@ -12,7 +12,7 @@ type TypedFloatArray = Float32Array | Float64Array;
 
 type TypedArray = TypedIntArray | TypedFloatArray;
 
-export type AnyArrayType = Array<any> | TypedIntArray | TypedFloatArray;
+export type AnyArrayType = Array<any> | TypedIntArray | TypedFloatArray | TypedArray;
```

Comment:
Does this actually fix anything? Regardless, you should be able to remove `TypedIntArray | TypedFloatArray` since they are the same as `TypedArray`?